lib/sms.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class SMS
+  attr_reader :credentials, :phone_number
+
+  def initialize
+    @credentials = Dotenv.load
+    @client = Twilio::REST::Client.new(credentials["TWILIO_ACCOUNT_SID"], credentials["TWILIO_AUTH_TOKEN"])
```

Comment:
Metrics/LineLength: Line is too long. [107/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)